anyway/parsers/registered.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+import csv
+import glob
+import io
+import logging
+import os
+import re
+import sys
+from datetime import datetime
+
+import six
+from flask.ext.sqlalchemy import SQLAlchemy
+
+from ..models import RegisteredVehicle, City
+from ..utilities import init_flask, time_delta
+
+reload(sys)
+sys.setdefaultencoding('utf8')
```

Comment:
This method doesn't exist in Python 3

Reply:
i know, i think you do not need it in py3 - actually it comes with the reload(sys) hack - it will go with the solution of reload(sys) hack, please advice.

Reply:
@yuvalkuck please take a look at the [utilities](https://github.com/hasadna/anyway/blob/dev/anyway/utilities.py) Roey wrote for working with utf-8, and how he's using them in [cbs.py](https://github.com/hasadna/anyway/blob/dev/anyway/parsers/cbs.py). Specifically, I think the functions `open_utf8` and `decode_hebrew` are relevant. Note how he's using `six` to determine whether the Python version is 2 or 3. If the existing utilities are not what you need, perhaps you could do something similar.

Reply:
ok, using your clue i was able to remove the need of thous lines.